Update feed docs (#10881)
forem/forem: docs/technical-overview/feed.md

# "The feed"

The Forem feed is perhaps _the_ core functionality of the service. It's an
adaptation of a concept popularized by other social platforms in the past two
decades, and it is something we need to develop with care in a way that empowers
individual communities and users.

The core nature of "the feed" is that it needs to evolve and be flexible. We
will learn new ideas over time. We need to take into account metrics, but also
question the relevance and "goodness" of certain metrics. We want folks finding
fulfilling and enriching content, not necessarily addictive content.

However, we are in the fairly naive early days of the feed, so primarily it is a
matter of flexibility and experimentation.

### Feed style

Each Forem can have a feed style set by the admin of that community (originally

> implemented [here](https://github.com/forem/forem/pull/8721)).

Reviewer: **Non-blocking**
```suggestion
implemented [in a PR from mid-2020](https://github.com/forem/forem/pull/8721)).
```

> Currently we have to styles: `rich` and `basic`. `rich` will always show the

Reviewer: A couple of suggestions and a typo:

```suggestion
Currently, we have two styles: `rich` and `basic`. Where `rich` always displays the
```

> cover image in the feed, `basic` will not.

Reviewer: ```suggestion
cover image in the feed, `basic` hides it from the feed view.
```

Over time, we'll continue to expand on what a `rich` feed and more minimal feed

> means and looks like.

Reviewer: **Non-blocking**

I think these two lines read kind of funny. We could drop it or make it more ambiguous? What do you think?

Something like: `Over time, we'll improve and expand our feed style options.` seems to give us more flexibility.

### Feed strategy

Each Forem can have a feed strategy set by the admin of that community

> (originally implemented [here](https://github.com/forem/forem/pull/10245)).

Reviewer: **Non-blocking**

Similar to my above suggestion regarding links and the word "here."

```suggestion
(originally implemented [in a late-2020 PR](https://github.com/forem/forem/pull/10245)).
```

> Currently we have two strategies: `basic` and `large_forem_experimental`... The

Reviewer: I'm not sure we should use an ellipsis, and we could put a comma in this sentence!

```suggestion
Currently, we have two strategies: `basic` and `large_forem_experimental`. The
```

"experimental" component dictates that there is some split testing, but
generally these are just cues for an underlying algorithm which can change
liberally.

The feed endpoint is driven by the `feeds_controller` and the content is found

> in objects such as `Articles::Feeds::Basic` etc. We should lean towards

Reviewer: **Non-blocking**

I think `etc` confuses me a bit here, but isn't a super strong opinion.

Also, I believe it should be `toward` rather than `towards`, but it is more of an opinion than a rule.

```suggestion
in objects such as `Articles::Feeds::Basic`. We should lean toward
```

> adaptation and versatility in the longrun here, even if we are just at the

Reviewer: **Non-blocking**

I think adaptability is the correct word in this case, but :shrug: I can go either way.

```suggestion
adaptability and versatility in the long run here, even if we are just at the
```
